xbmc/cores/AudioEngine/Sinks/AESinkALSA.h

```diff
@@ -34,6 +34,8 @@
 // being destructed before CALSA*Monitor below.
 #include "platform/linux/FDEventMonitor.h"
 
+#define AE_MIN_PERIODSIZE 256
```

Comment:
defines in header are ugly. if you want to move it, make it a const and give it class context